### DISCUSSION TITLE
calendar icon showing up next to easepick calendar ??

### DISCUSSION THREAD

**Discussion by gleysen** [easepick/easepick#204] [Q&A] [answered]
When I insert the easepick calendar, there is a small calendar icon showing up next to it... when I see in the source it shows:

```
<style>
      button {
        border: none;
        background: transparent;
        font-size: 18px;
      }
</style>
<button>📅</button>
```

Where is this coming from?!
And how do I remove/hide it?
Thx!

Geert

**Comment by wakirin** [comment, marked as answer]
its from kbd-plugin, but no option to hide it.
try to hide it with css styles.

**Reply by dagassa** [reply]
hi @wakirin 

```
:host(button) {
	display: none;
  }
```

I've tried numerous approaches but still can't select this button. It remains stuck in the UI  somewhere when the inline calendar is activated and defies attempts to hide it with CSS due to its encapsulation within its own style tag in the shadow DOM. I would like to know if it could be hidden using an ID, a class name, or a data attribute, but none of these methods have succeeded.
 
this makes the kbd-plugin unusable.

🙏 Thanks

**Comment by gleysen** [comment]
Great, thx! (I just removed the kbd-plugin since I don't really need that ;-)
